src/sentry/tsdb/redis.py

```diff
@@ -485,16 +485,17 @@ def get_frequency_series(self, model, items, start, end=None, rollup=None):
 
         # Freeze ordering of the members (we'll need these later.)
         for key, members in items.items():
-            items[key] = tuple(members)
+            items[key] = list(members)
```

Comment:
It's not clear to me why we need this extra-first-pass-loop instead of just doing the `list(members)` on line 498 to change `arguments + members` to `arguments + list(members)`?